x-pack/plugins/secops/public/store/local/timeline/reducer.ts

```diff
@@ -218,10 +298,6 @@ export const timelineReducer = reducerWithInitialState(initialTimelineState)
     ...state,
     timelineById: removeTimelineProvider({ id, providerId, timelineById: state.timelineById }),
   }))
-  .case(updateData, (state, { id, data }) => ({
```

Comment:
note: I removed this on the suggestion from @andrew-goldstein as this is dead and no longer used as we use apollo graphql and apollo client.

Reply:
I think this is what we hoped for: `graphql` is reducing the need for state in `redux`. 👍